.gitlab-ci.yml

```diff
@@ -0,0 +1,64 @@
+stages:
+  - test
+  - build
+
+image:                             parity/rust:nightly
+
+variables:
+  CI_SERVER_NAME:                  "GitLab CI"
+  CARGO_HOME:                      "${CI_PROJECT_DIR}/.cargo"
+
+
+cache:
+  key:                             "${CI_JOB_NAME}"
+  paths:
+    - ./target/
+    - ./.cargo/
+
+
+.collect_artifacts:                &collect_artifacts
```

Comment:
Do we also need `.releasable_branches` configs if we enabled `.collect_artifacts`?